Feat/social event

alexsavio opened the pull request:
This PR adds a page for the social event.

datapythonista commented:
Are the added slugs changing the urls of the pages? I'd prefer to not change the urls at this point if that's the case, we'll break links in search engines, social media...

Also, did you check how the menu renders? The space for the sections is limited, and I think it can overflow with the new "Social event" section. I think my preference is to add a link in the program, and may be somewhere in the home, and not have a section for it in the top menu.

alexsavio commented:
As far as I see, the slugs are not used anywhere, I just added them so the other ones wouldn't feel alone. I would have actually deleted them because AFAIK the `save_as` field is what defines the file name, hence the urls. The slugs are not used in the setup for this year, we are not using templates or articles.

I removed the social event from the menu, added the url in the index and program pages.

Thanks for the review!
